go/libkb/secret_store_file.go

```diff
@@ -88,6 +215,20 @@ func (s *SecretStoreFile) StoreSecret(username NormalizedUsername, secret LKSecF
 }
 
 func (s *SecretStoreFile) ClearSecret(username NormalizedUsername) error {
+	// try both
+	errV1 := s.clearSecretV1(username)
+	errV2 := s.clearSecretV2(username)
+
+	if errV1 != nil {
+		return errV1
+	}
+	if errV2 != nil {
+		return errV2
+	}
+	return nil
+}
+
+func (s *SecretStoreFile) clearSecretV1(username NormalizedUsername) error {
```

Comment:
I think it still makes sense to shred the V1 file, since it might do something useful on magnetic disks.